java/json/src/main/java/org/bondlib/JsonGlobals.java

```diff
@@ -0,0 +1,18 @@
+package org.bondlib;
+
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.core.JsonParser;
+
+/**
+ * Contains shared global references and constants.
+ */
+final class JsonGlobals {
+
+    // thread-safe shared factory for parsers
+    static final JsonFactory jsonFactory = new JsonFactory();
```

Comment:
Jackson recommends that. JsonFactory is heavyweight but thread-safe.

---
In reply to: [143811425](https://github.com/Microsoft/bond/pull/632#discussion_r143811425) [](ancestors = 143811425)

Reply:
I'd like a comment here explaining that we are doing this because jackson recommends it, and ideally, a link to the documentation where they do. Either way, I would like to read what you read.